<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QWidget" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grub-on</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="timeoutBox">
         <property name="suffix">
          <string/>
         </property>
         <property name="prefix">
          <string>Zaman Aşımı : </string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="hiddenMenuCheckBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Menüyü Gizle</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QListWidget" name="osListWidget"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QToolButton" name="upButton">
           <property name="minimumSize">
            <size>
             <width>79</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Yukarı</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="downButton">
           <property name="minimumSize">
            <size>
             <width>79</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Aşağı</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="setDefaultButton">
           <property name="minimumSize">
            <size>
             <width>79</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Varsayılan Yap</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="addButton">
           <property name="minimumSize">
            <size>
             <width>79</width>
             <height>0</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Ekle</string>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeButton">
           <property name="minimumSize">
            <size>
             <width>79</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Kaldır</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="osOptionsEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="backupButton">
       <property name="text">
        <string>Yedekle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restoreButton">
       <property name="text">
        <string>Yedeği Yükle</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>37</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Sıfırla</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Kaydet</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
